docs/release-notes/8.15.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[release-notes-header-8.15.0]]
+== 8.15.0
+
+[discrete]
+[[release-notes-8.15.0]]
+=== 8.15.0
+
+[discrete]
+[[breaking-changes-8.15.0]]
+==== Breaking changes
+
+* User-defined quick prompts for Elastic AI Assistant that were created before you upgrade to 8.15 will no longer appear after you upgrade. In 8.15, quick prompts will be shared by all users in your deployment, rather than stored at the user level. To implement your existing quick prompts in 8.15, copy them prior to upgrade, and add them again after upgrade ({kibana-pull}187040[#187040]).
+
+[discrete]
+[[features-8.15.0]]
+==== New features
+
+* Adds the `scan` action to the response console, which allows you to scan a specific file or directory on a host for malware ({kibana-pull}184723[#184723]).
+* Adds a Defend Advanced Policy option that allows you to opt out of registry event filtering ({kibana-pull}186564[#186564]).
+* Allows you to specify additional file and registry paths to monitor for read access ({kibana-pull}181361[#181361]).
+* Allows you to use {elastic-sec} to isolate and release hosts running a Crowdstrike agent ({kibana-pull}186801[#186801]).
+* Updates entity risk scores when asset criticality changes ({kibana-pull}182234[#182234]).
```

Comment:
```suggestion
* Recalculates entity risk scores when asset criticality changes on an individual entity ({kibana-pull}182234[#182234]).
```
Since this doesn't apply to bulk assignment yet.